src/functions/template-tags/general.php

```diff
@@ -1344,8 +1322,24 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 		 */
 		$excerpt_more = apply_filters( 'excerpt_more', ' [&hellip;]' );
 
-		// Now we actually trim it
-		$excerpt = wp_trim_words( $excerpt, $excerpt_length, $excerpt_more );
+		// Trim the excerpt. The following is just wp_trim_words() without its use of wp_strip_all_tags().
```

Comment:
> The following is just wp_trim_words() __without its use of wp_strip_all_tags()__

So ... I realize we can't strip tags __and__ preserve HTML - but looking through the remainder of the function there doesn't seem to be anything to handle elements that are chopped in two. For example, let's say the cut off point is half way through the link here:

```html
Blah blah apples and <a href="...">look here</a> to learn about oranges.
                                       ^ 
                                 Cutoff point
```

Aren't we going to end up with an unclosed element?